src/game/Tactical/LOS.cc

```diff
@@ -4311,15 +4307,10 @@ INT32 CheckForCollision(FLOAT dX, FLOAT dY, FLOAT dZ, FLOAT dDeltaX, FLOAT dDelt
 	sX = (INT16)( dX / CELL_X_SIZE );
 	sY = (INT16)( dY / CELL_Y_SIZE );
 
-	// Check if gridno is in bounds....
-	if ( !GridNoOnVisibleWorldTile( (INT16) (sX + sY * WORLD_COLS) ) )
-	{
-		//return( COLLISION_NONE );
-	}
-
-	if ( sX < 0 || sX > WORLD_COLS || sY < 0 || sY > WORLD_COLS )
+	if (sX < 0 || sX >= WORLD_COLS || sY < 0 || sY >= WORLD_COLS)
```

Comment:
One of these should be WORLD_ROWS, I guess the last.

Reply:
Yes, I didn't even notice that one.